src/manifest.ts

```diff
@@ -337,11 +338,12 @@ export class Manifest {
     getFileMethod: 'getConfigJson' | 'getManifestJson',
     fileName: string
   ): Promise<{valid: true; obj: object} | {valid: false; obj: undefined}> {
-    let response: {valid: true; obj: object} | {valid: false; obj: undefined} =
-      {
-        valid: false,
-        obj: undefined,
-      };
+    let response:
+      | {valid: true; obj: object}
```

Comment:
something weird going on here. This annotation claims it should be back in its original form. but in its original form gts complains and says that prettier wants it new-lined. ♾️

Reply:
btw this is how `npm run fix` leaves it

Reply:
You can try `rm -rf package-lock.json` and `rm -rf node_modules`, then `npm i` and `npm run fix`, see if that fixes your linting issue.

Reply:
oh interesting, after doing that `npm run fix` puts it back in the original form and `npm run lint` no longer complains about it. thanks for the tip!